[2025-02-11T07:56:08.284+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-11T07:56:08.333+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.extract.extract_and_load_to_staging scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T07:56:08.352+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.extract.extract_and_load_to_staging scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T07:56:08.353+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-11T07:56:08.374+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract.extract_and_load_to_staging> on 2025-02-10 00:00:00+00:00
[2025-02-11T07:56:08.382+0000] {standard_task_runner.py:63} INFO - Started process 201 to run task
[2025-02-11T07:56:08.388+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'extract.extract_and_load_to_staging', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/e_commerce_dw_dag.py', '--cfg-path', '/tmp/tmpv4ulpluw']
[2025-02-11T07:56:08.392+0000] {standard_task_runner.py:91} INFO - Job 185: Subtask extract.extract_and_load_to_staging
[2025-02-11T07:56:08.453+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.extract.extract_and_load_to_staging scheduled__2025-02-10T00:00:00+00:00 [running]> on host 1f90b0c5673e
[2025-02-11T07:56:08.568+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='extract.extract_and_load_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-11T07:56:08.569+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-11T07:56:08.604+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-11T07:56:08.819+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-11T07:56:08.922+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-02-11T07:56:08.999+0000] {logging_mixin.py:188} INFO -   id   name
0  1  thang
1  2    lam
2  3   tuan
[2025-02-11T07:56:09.001+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-11T07:56:09.003+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-11T07:56:09.019+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=extract.extract_and_load_to_staging, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250211T075608, end_date=20250211T075609
[2025-02-11T07:56:09.083+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-11T07:56:09.112+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-11T07:56:09.115+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
